README.md

```diff
@@ -157,6 +157,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Shikimori](https://shikimori.one/api/doc) | Anime discovery, tracking, forum, rates | `OAuth` | Yes | Unknown |
 | [Studio Ghibli](https://ghibliapi.herokuapp.com) | Resources from Studio Ghibli films | No | Yes | Yes |
 | [Trace Moe](https://soruly.github.io/trace.moe-api/#/) | A useful tool to get the exact scene of an anime from a screenshot | No | Yes | No |
+| [Waifu.im](https://waifu.im/docs) | Get waifu pictures from an archive of over 4000 images and multiple tags | No | Yes | No |
```

Comment:
The site is down. Please fix it and let me know

Reply:
Yes, I was reworking it a bit and had to go to sleep before finishing, I'll let you know when it's back. Sorry for the inconvenience.

Reply:
sure, no problem.

Reply:
It's normally back, sorry for the wait.